resources/views/welcome.blade.php

```diff
@@ -66,6 +66,16 @@
             .m-b-md {
                 margin-bottom: 30px;
             }
+
+            .error {
```

Comment:
can we make these message--error and message--success and make the hex codes lowercase?